ast/policy.go

```diff
@@ -1479,6 +1491,21 @@ func (q *Every) Compare(other *Every) int {
 	return q.Body.Compare(other.Body)
 }
 
+// Vars returns the key and val arguments of an every expression,
+// if they are non-nil and not wildcards.
+func (q *Every) Vars() VarSet {
```

Comment:
nit: could we call this something else? Perhaps `KeyValueVars()`. The reason is that we have `Vars()` functions on many other types and normally they either return ALL vars under the AST node or they allow callers to supply a parameter struct to customize it. This function diverges a bit by only looking at key/value.